x-pack/test/functional/es_archives/saved_objects_management/spaces_integration/mappings.json

```diff
@@ -0,0 +1,473 @@
+{
+  "type": "index",
+  "value": {
+    "index": ".kibana",
+    "settings": {
+      "index": {
+        "number_of_shards": "1",
+        "auto_expand_replicas": "0-1",
+        "number_of_replicas": "0"
+      }
+    },
+    "mappings": {
+      "dynamic": "strict",
+      "properties": {
+        "apm-telemetry": {
```

Comment:
😞

Reply:
yea... and it is actually necessary, removing it show errors in the FTR server logs sometimes...